cpp/tests/column/compound_test.cu

```diff
@@ -84,19 +84,25 @@ TEST_F(CompoundColumnTest, ChildrenLevel1)
         cudf::data_type{cudf::STRING}, 100, rmm::device_buffer{0},
         rmm::device_buffer{0}, 0, std::move(children));
 
-  auto column = cudf::column_device_view::create(parent->view());
-  auto d_column = *column;
-
-  EXPECT_FALSE(d_column.nullable());
-  EXPECT_FALSE(d_column.has_nulls());
-  EXPECT_EQ(0, d_column.null_count());
-
-  bool check = thrust::any_of( thrust::device, thrust::make_counting_iterator<int32_t>(0),
-        thrust::make_counting_iterator<int32_t>(100), checker_for_level1{d_column} );
-  EXPECT_TRUE(check);
+  {
+      auto column = cudf::column_device_view::create(parent->view());
+      EXPECT_FALSE(column->nullable());
+      EXPECT_FALSE(column->has_nulls());
+      EXPECT_EQ(0, column->null_count());
```

Comment:
FYI, @nvdbaranec is working on removing `null_count` and `has_nulls` from `column_device_view` in issue https://github.com/rapidsai/cudf/issues/4368

Reply:
Yeah, I know. I don't think these are really needed here anyway. I'll just remove them.